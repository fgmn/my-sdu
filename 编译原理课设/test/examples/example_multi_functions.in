procedure f1;
begin
end;
procedure f2;
begin
end;
procedure f3;
begin
end;
procedure f4;
begin
end;
.