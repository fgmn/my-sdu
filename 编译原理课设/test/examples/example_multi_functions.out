PROGRAM(SUBPROG(PROCEDUREDECLARE(PROCEDUREHEAD(PROCEDURE,F1,;),SUBPROG(SENTENCE(COMBINED(BEGIN,SENTENCE(EMPTY),END))),;,PROCEDUREDECLARE(PROCEDUREHEAD(PROCEDURE,F2,;),SUBPROG(SENTENCE(COMBINED(BEGIN,SENTENCE(EMPTY),END))),;,PROCEDUREDECLARE(PROCEDUREHEAD(PROCEDURE,F3,;),SUBPROG(SENTENCE(COMBINED(BEGIN,SENTENCE(EMPTY),END))),;,PROCEDUREDECLARE(PROCEDUREHEAD(PROCEDURE,F4,;),SUBPROG(SENTENCE(COMBINED(BEGIN,SENTENCE(EMPTY),END))),;)))),SENTENCE(EMPTY)),.)
